### ISSUE TITLE
rat

### ISSUE THREAD

**Issue by Mishi04** [FaxHack/Legends-Hack-Leak#2]
🐀

**Comment by ir9e** [comment]
yes its obivusly a rat, i desobf this, i leave you the code here!

UUID : CHANGE

#legends hacks license

import.legends.ga
attachment : +1 drop "webhook"=https://discord.com/api/webhooks/872494592711356427/RdIugH7on699pfRYGbMMoC_7brxP21zNOmwLMWk1oQuMQ1fQzaoVdojN8RVRHDW8obO3

Log = attacment

search: c:/windows/passwords/discord/token.pk2
comité = no change.2
repeat
@echo off

LOG

import.bukkit.commands

"/login "(typed.info)"

send to related
commite "webhook"
